content/docs/sidebar.json

```diff
@@ -72,6 +72,10 @@
         "slug": "model-management",
         "source": "model-management/index.md",
         "children": [
+          {
```

Comment:
Let's make this the second item after `Model registry`, not before.

Reply:
Not sure I follow. I don't think it will make sense to show how to do all the MR actions (register, assign, etc.) and only show how to add a model afterwards. Is it the order of the content that's confusing, or is it the page titles? Maybe we should call the whole section Model Registry and call the second page Model Management?

Reply:
> Is it the order of the content that's confusing, or is it the page titles?

A bit of both.

> Maybe we should call the whole section Model Registry and call the second page Model Management?

Makes sense. I think we need to slightly update the pages' content also. I'll propose the changes shortly for you to review.

Reply:
[PR](https://github.com/iterative/dvc.org/pull/5144)